### PR TITLE
fix destroy function

### PR DESCRIPTION
**Pull request by nickcuper**
This change is [Reviewable](https://reviewable.io/reviews/openplayerjs/openplayerjs/111)

### CODE REVIEW

File: src/js/player.ts [hunk 1]
```diff
@@ -400,6 +400,7 @@ class Player {
         }
 
         el.controls = true;
+        el.setAttribute('id', this.uid);
```
Comment (rafa8626) [inline src/js/player.ts]: Why is this needed when we destroy the player?

Comment (nickcuper) [reply src/js/player.ts]: When player has been created you remove id from video element
For case when i want to recreate player i should do 
```js
player.destroy();
// you will got problem here cuz current video element does not have id attribute
player = new Openplayer('videoElementId'); 
```
I added test to cover this case

**Comment by rafa8626** [comment]
Gotcha; I never considered that scenario. I’ll merge it then, and thanks for collaborating with this project

File: src/js/player.ts (hunk 1 above)
Comment (nickcuper) [reply src/js/player.ts]: actually problem is here (in constructor, first line)
```js
this.element = element instanceof HTMLMediaElement ? element : (document.getElementById(element) as HTMLMediaElement); // return null after destroy
```

```js
constructor(element: HTMLMediaElement | string, ads?: string | string[], fill?: boolean, options?: PlayerOptions) {
       // check this line
        this.element = element instanceof HTMLMediaElement ? element : (document.getElementById(element) as HTMLMediaElement);
        if (this.element) {
            this.ads = ads;
            this.fill = fill;
            this.autoplay = this.element.autoplay || false;
            this.options = deepmerge(this.defaultOptions, options || {});
            this.element.volume = this.options.startVolume;
            if (this.options.startTime > 0) {
                this.element.currentTime = this.options.startTime;
            }
            this.volume = this.element.volume;
        }
        return this;
    }
```

**Comment by rafa8626** [comment]
@nickcuper merged; let me know if you plan to provide another PR so I can hold off before I deploy the next release tomorrow
